extractor/filesystem/secrets/convert/specific.go

```diff
@@ -0,0 +1,94 @@
+package convert
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/osv-scalibr/extractor/filesystem"
+	"github.com/google/osv-scalibr/inventory"
+	"github.com/google/osv-scalibr/plugin"
+	"github.com/google/osv-scalibr/veles"
+)
+
+// FromVelesDetectorWithRequire converts a Veles Detector into a SCALIBR FilesystemExtractor plugin.
+// This allows:
```

Comment:
These don't bullet points seem specific to this function - all of these apply to FromVelesDetector() as well. 
How about we just say something like `FromVelesDetectorWithRequire works similar to FromVelesDetector but allows specifying additional files to look at on top of the default ones.`